quantum/rgb_matrix/rgb_matrix.c

```diff
@@ -249,8 +249,15 @@ void process_rgb_matrix(uint8_t row, uint8_t col, bool pressed) {
 #endif // RGB_MATRIX_KEYREACTIVE_ENABLED
 
 #if defined(RGB_MATRIX_FRAMEBUFFER_EFFECTS) && defined(ENABLE_RGB_MATRIX_TYPING_HEATMAP)
-    if (rgb_matrix_config.mode == RGB_MATRIX_TYPING_HEATMAP) {
-        process_rgb_matrix_typing_heatmap(row, col);
+#    if defined(RGB_MATRIX_KEYRELEASES)
+    if (!pressed)
+#    else
+    if (pressed)
+#    endif // defined(RGB_MATRIX_KEYRELEASES)
+    {
```

Comment:
Not a huge fan of this.  This means that if you define both release and press, it will still only trigger on one of them.

Reply:
It hadn't occurred to me that you might want to trigger it both ways. But I believe this follows the same behaviour as the other reactive effects. https://github.com/qmk/qmk_firmware/blob/aa06893b58ac3ed88e796968d237f60f4c156875/quantum/rgb_matrix/rgb_matrix.c#L224-L228

Reply:
Ah, fair.  In this case, would be outside of the scope of this PR, I'd say.